core/types/src/account.rs

```diff
@@ -0,0 +1,163 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
+//! Mobilecoin Account and Subaddress objects
+
+use zeroize::Zeroize;
+
+use crate::keys::{
+    RootSpendPrivate, RootSpendPublic, RootViewPrivate, RootViewPublic, SubaddrSpendPrivate,
+    SubaddrSpendPublic, SubaddrViewPrivate, SubaddrViewPublic,
+};
+
+/// An object which represents a subaddress, and has RingCT-style
```

Comment:
:question: Is the same trait in `mc-account-keys` going to go away in a subsequent PR?

Reply:
ahh yes, in my branch the `mc-account-keys` version is just a re-export of this but i seem to have missed pulling it in / might as well just drop it i suspect.